__tests__/server.test.js

```diff
@@ -40,3 +41,26 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET:200 responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
+          expect(article.body).toBe(undefined);
+        });
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+
```

Comment:
To make sure that you've received all the articles, you will need to test the length of the array as well